dev/request-logging-dev.xml

```diff
@@ -0,0 +1,8 @@
+<configuration scan="true" debug="false">
+  <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
+    <encoder>
+        <pattern>%h %l %u %user %date "%r" %s %b %h %a %localPort %D</pattern>
+    </encoder>
+  </appender>
+  <appender-ref ref="STDOUT" />
+</configuration>
```

Comment:
Pretty much, except I hadn't been thinking we would dump that stuff to STDOUT.  I think I was thinking maybe we'd use a file appender, but hadn't thought much about where the file would go.  Maybe just in like './target/logs/puppetserver-access.log' or something?